src/geotools/src/test/java/org/locationtech/geogig/geotools/data/GeoGigDataStoreFactoryTest.java

```diff
@@ -30,12 +30,9 @@
 
     private GeoGigDataStoreFactory factory;
 
-    private File repoDirectory;
-
     @Override
     protected void setUpInternal() throws Exception {
         factory = new GeoGigDataStoreFactory();
-        repoDirectory = super.repositoryDirectory;
```

Comment:
I just removed this local attribute as it is the same thing as the parent attribute "repositoryDirectory".